js/components/ChartForm.vue

```diff
@@ -567,9 +567,7 @@ export default {
           }
         };
 
```

Comment:
The [heuristic used to break call chains was updated](https://prettier.io/blog/2020/03/21/2.0.0.html#improved-method-chain-breaking-heuristic-6685httpsgithubcomprettierprettierpull6685-by-mmkalhttpsgithubcommmkal) to produce less wrapping.